packages/tupaia-web-server/src/routes/EntityAncestorsRoute.ts

```diff
@@ -23,10 +24,18 @@ export class EntityAncestorsRoute extends Route<EntityAncestorsRequest> {
     const { rootEntityCode, projectCode } = params;
     const { includeRootEntity = false, ...restOfQuery } = query;
 
+    const project = (
+      await ctx.services.central.fetchResources('projects', {
+        filter: { code: projectCode },
+        columns: ['config'],
+      })
+    )[0];
+    const { config } = project;
+
     const entities = await ctx.services.entity.getAncestorsOfEntity(
       projectCode,
       rootEntityCode,
-      { fields: DEFAULT_FIELDS, ...restOfQuery },
+      { filter: generateFrontendExcludedFilter(config), fields: DEFAULT_FIELDS, ...restOfQuery },
```

Comment:
I'm not entirely sure if this is useful in the ancestors route, I _think_ we're usually excluding types without any children. But I've added it here for consistency

Reply:
I'd be interested to know if things works when we exclude here. I'd expect them to not work, but that's fine. It'd basically break the hierarchy. Keep it here though, that way the faulty config will fail noisily rather than silently 👍